client/plots/singleCellPlot.js

```diff
@@ -798,7 +818,11 @@ class singleCellPlot {
 
 	getOpacity(d) {
 		if (this.config.hiddenClusters.includes(d.category)) return 0
-		return 0.8
+		if (this.colorByGene && this.state.config.gene) {
+			if (this.state.config.min > d.geneExp) return 0
+			if (this.state.config.max < d.geneExp) return 0
```

Comment:
wrong. use max color if gene exp value > max.

Reply:
I am not sure I understand. I need to troubleshoot the profile now but maybe we can talk later

Reply:
I am commenting out the setMinMaxCallback for now if is ok. This was a minor change in this PR and I can focus on this issue in a next PR.